src/client/client.cpp

```diff
@@ -1768,11 +1768,12 @@ void Client::makeScreenshot()
 	if (!raw_image)
 		return;
 
-	time_t t = time(NULL);
-	struct tm *tm = localtime(&t);
+	struct tm tm;
+	const time_t t = time(NULL);
+	localtime_r(&t, &tm);
```

Comment:
`localtime_r` does not exist on Windows so this breaks the build
I also don't know if it exists on Android (CI currently doesn't work)

Reply:
Should we use `localtime_s` (which does exist on Windows (and probably Android))?

Reply:
Also see: https://en.cppreference.com/w/c/chrono/localtime

Reply:
I tried using `localtime_s` now, and all the checks fail now. The other checks are saying that `localtime_s` is not declared in this scope. I'll use `localtime_r` for now. 🤔